GliaWidgets/Sources/View/Common/SelfSizingCollectionView/SelfSizingCollectionView.swift

```diff
@@ -27,6 +27,7 @@ final class SelfSizingCollectionView: UICollectionView {
     }
 
     override var intrinsicContentSize: CGSize {
+//        self.layoutIfNeeded()
```

Comment:
Probably not needed

Reply:
Fixed 4285cdde0a5c765ee09b36d30c2a36d0bfc1b777